.gitignore

```diff
@@ -0,0 +1,12 @@
+/.idea/
+node_modules/
+/.parcel-cache/
+/client/.parcel-cache/
+/client/__tests__/__snapshots__/
+/server/__tests__/__snapshots__/
```

Comment:
snapshots skal ikke gitignores!

- [ ] fjern snapshots fra .gitignore
- [ ] sjekk inn snapshots

Reply:
> Bra saker! Her er dere på rett veil. Sjekkliste for hva dere må beherske
> 
> * [ ]  `Simulate` og `jest.fn` i klient tester
> * [ ]  `supertest` i server tester
> * [ ]  husk å git-ignore .env-fila
> * [ ]  _ikke_ git-ignore snapshots

Takk for tilbakemeldinger!